cionic/kinematics.py

```diff
@@ -675,77 +757,129 @@ def stat_tests(self, k, groups, planes, split):
                 scoretable['Max_Full'] = max_full
                 scoretable['Min_Full'] = min_full
                 scoretable['MaxC_Full'] = maxc_full
                 scoretable['MinC_Full'] = minc_full
                 scoretable['ROM_Full'] = rom_full
-        
-        listnames, pvals = [],[]
+
+        listnames, pvals = [], []
         for c in ['Max_Full', 'Min_Full', 'MaxC_Full', 'MinC_Full', 'ROM_Full']:
             for a in scoretable.Angle.unique():
-                right = scoretable.loc[(scoretable['Angle'] == a) & (scoretable['Side'] == 'right'), c].values[0]
-                left =  scoretable.loc[(scoretable['Angle'] == a) & (scoretable['Side'] == 'left'), c].values[0]
-                s,p = ttest_ind(np.squeeze(right),np.squeeze(left), equal_var=False)
-                listnames.append(a+'_'+c)
+                right = scoretable.loc[
+                    (scoretable['Angle'] == a) & (scoretable['Side'] == 'right'), c
+                ].values[0]
+                left = scoretable.loc[
+                    (scoretable['Angle'] == a) & (scoretable['Side'] == 'left'), c
+                ].values[0]
+                s, p = ttest_ind(np.squeeze(right), np.squeeze(left), equal_var=False)
+                listnames.append(a + '_' + c)
                 pvals.append(p)
 
         outtable = pd.DataFrame()
         outtable['Score'] = listnames
         outtable['P-value'] = pvals
-        
+
         return outtable
 
-    
-    def asi(self,a,b):
-        si = (100 - (abs((a-b)/(0.5*(a+b)))*100))
+    def asi(self, a, b):
+        si = 100 - (abs((a - b) / (0.5 * (a + b))) * 100)
         if si < 0:
             si = 0
         return si
-    
-    
+
     def normgaitscore(self, gaitscores, norm_kin_data_dict, groups, planes):
         normdf = pd.DataFrame()
-        angles, sides, Nmax, Nmin, NmaxC, NminC, NROM = [],[],[],[],[],[],[]
+        angles, sides, Nmax, Nmin, NmaxC, NminC, NROM = [], [], [], [], [], [], []
         for side in groups:
             for angle in planes:
-                normdf[(side + '_' + angle)] = norm_kin_data_dict[side][angle]['avg_angle']
+                normdf[(side + '_' + angle)] = norm_kin_data_dict[side][angle][
+                    'avg_angle'
+                ]
                 angles.append(angle)
                 sides.append(side)
-                Nmax.append(round(normdf[(side + '_' + angle)][51:].max(),2))
-                NmaxC.append(normdf[(side + '_' + angle)][normdf[(side + '_' + angle)] == normdf[(side + '_' + angle)][51:].max()].index[0])
-                Nmin.append(round(normdf[(side + '_' + angle)][51:].min(),2))
-                NminC.append(normdf[(side + '_' + angle)][normdf[(side + '_' + angle)] == normdf[(side + '_' + angle)][51:].min()].index[0])
-                NROM.append(round(abs((normdf[(side + '_' + angle)][51:].max()) - (normdf[(side + '_' + angle)][51:].min())),2))
-
+                Nmax.append(round(normdf[(side + '_' + angle)][51:].max(), 2))
```

Comment:
nit (not needed in this PR): magic numbers like `51` here make me nervous - could it be a const instead?